### PR TITLE
Feature/sidebar

### PR DESCRIPTION
**Pull request by OlesiaTr**

### CODE REVIEW

File: cms/src/components/Button/Button.styles.ts [hunk 1]
```diff
@@ -1,5 +1,8 @@
 import styled from '@emotion/styled';
 
+import { colors, fonts, media } from '../../styles/theme';
+import { IconSprite } from '../IconSprite/IconSprite';
```
Comment (MaksVibe) [inline cms/src/components/Button/Button.styles.ts]: Do you use IconSprite here?

File: cms/src/components/Header/Header.tsx [hunk 2]
```diff
@@ -20,28 +27,56 @@ import {
...
-      <MobileHeaderWrapper>
-        <MenuWrapper>
-          <MenuBtn onClick={() => console.log('Open menu')}>
-            <MenuIcon />
-          </MenuBtn>
-        </MenuWrapper>
-        <SearchWrapper>
+
+      <MobileHeaderWrapper isOpen={showModal}>
+        {/*Why it doesnt work?*/}
```
Comment (MaksVibe) [inline cms/src/components/Header/Header.tsx]: what?

File: cms/src/components/Header/Header.tsx [hunk 3]
```diff
@@ -20,28 +27,56 @@ import {
...
+        {mobile ||
+          (windowWidth && windowWidth < 768 && (
+            <MenuWrapper>
+              <MenuBtn onClick={toggleSidebar}>{showSidebar ? <CloseMenuIcon /> : <MenuIcon />}</MenuBtn>
+            </MenuWrapper>
+          ))}
+        <SearchWrapper isOpen={showModal}>
           <SearchInput placeholder="Ключове слово" maxLength={50} onInput={(ev) => console.log(ev.target.value)} />
           <SearchIconBox>
```
Comment (MaksVibe) [inline cms/src/components/Header/Header.tsx]: ev.target.value has some type issue
Screenshot 2023-08-28 at 12 07 19

File: cms/src/components/Sidebar/SidebarModal.tsx
Comment (MaksVibe) [inline cms/src/components/Sidebar/SidebarModal.tsx:1]: fix the issues
Screenshot 2023-08-28 at 12 09 02

**Review by MaksVibe — CHANGES_REQUESTED** [review]

**Comment by MaksVibe** [comment]
Make linting and fix issues, pls

File: cms/src/components/Button/Button.styles.ts (hunk 1 above)
Comment (OlesiaTr) [reply cms/src/components/Button/Button.styles.ts]: Deleted!

File: cms/src/components/Header/Header.tsx (hunk 3 above)
Comment (OlesiaTr) [reply cms/src/components/Header/Header.tsx]: Done

File: cms/src/components/Header/Header.tsx (hunk 2 above)
Comment (OlesiaTr) [reply cms/src/components/Header/Header.tsx]: Got rid of mobile variable and it fixed itself

File: cms/src/components/Sidebar/SidebarModal.tsx
Comment (OlesiaTr) [reply cms/src/components/Sidebar/SidebarModal.tsx:1]: I dont have closeModal issue on my end

Comment (OlesiaTr) [reply cms/src/components/Sidebar/SidebarModal.tsx:1]: But I fixed return warning

**Review by MaksVibe — APPROVED** [review]

**Pull request merged**